scripts/quality/check_lock_paths.py

```
@@ -0,0 +1,83 @@
+    lock_paths_str = lock_paths_match.group(1).strip()
+    lock_paths = [p.strip() for p in lock_paths_str.split(",")]
+
+    return lock_paths
+
+
+def get_changed_files():
+    import subprocess
+
+    try:
+        result = subprocess.run(
+            ["git", "diff", "--name-only", "HEAD~1", "HEAD"],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)